src/main/java/roomescape/member/infra/MemberArgumentResolver.java

```diff
@@ -0,0 +1,42 @@
+
+package roomescape.member.infra;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+import roomescape.member.application.AuthService;
+import roomescape.member.dto.MemberRequestDto;
+
+import static roomescape.member.infra.CommonMethod.extractTokenFromCookie;
+
+@Component
+public class MemberArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private static final Logger log = LoggerFactory.getLogger(MemberArgumentResolver.class);
+    private final AuthService authService;
+
+    public MemberArgumentResolver(AuthService authService) {
+        this.authService = authService;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(LoginMember.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
```

Comment:
```suggestion
    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
```

`throws Exception` 부분은 불필요하므로 제거해보면 좋을 것 같아요.
그리고 굉장히 길어서 줄바꿈을 적용하면 읽기 더 편하겠네요!